DEPRECATED.md

```diff
@@ -60,8 +60,8 @@ N/A
 
 #### Announcement (Jaeger)
 
-* Version: TO-BE-RELEASED-VERSION
-* Date: TO-BE-RELEASED-DATE
+* Version: 1.8.2
+* Date: 2023-01-31
 * PR: [DEPRECATION] Deprecate the Jaeger exporter
   [#9999](https://github.com/open-telemetry/opentelemetry-cpp/pull/9999)
```

Comment:
Should the below PR URL be updated as well?

Reply:
Thanks, done!